cli/connect/console.go

```diff
@@ -186,6 +187,12 @@ func getExecutor(console *Console) prompt.Executor {
 			log.Debugf("Failed to append command to history file: %s", err)
 		}
 
+		evalFuncBody, err := static.GetStaticFileContent(ConnectLuaTemplateFS, "eval_func_body.lua")
```

Comment:
`evalFuncBody` can be set up once in `init()` function. In case of error we should panic.